app/src/main/java/org/p2p/wallet/newsend/api/GenerateTransactionResponse.kt

```diff
@@ -0,0 +1,53 @@
+package org.p2p.wallet.newsend.api
+
+import com.google.gson.annotations.SerializedName
+import java.math.BigDecimal
+import java.math.BigInteger
+
+data class GenerateTransactionResponse(
```

Comment:
GenerateSendTransactionResponse
we can generate transaction for a lot of things